src/EnergyPlus/HeatBalanceAirManager.cc

```diff
@@ -515,6 +515,41 @@ namespace HeatBalanceAirManager {
                 SetupOutputVariable(
                     "Zone Air Heat Balance Deviation Rate", OutputProcessor::Unit::W, ZnAirRpt(Loop).imBalance, "System", "Average", Zone(Loop).Name);
             }
+
+            if (!DisplayAdvancedReportVariables) {
```

Comment:
Did you intend to turn off these report variables if DisplayAdvancedReportVariables is true?

Reply:
@rraustad Yes. Actually this is a design decision I haven't thought through and was intended to discuss with reviewers. Currently we have a flag for the heat emission - output:table:tabular, but there's no global flag for heat emission - turn on additional report variables. 

Intermediate vars (such as surface level heat emission, zone level exfiltration and exhaust) are calculated and reported if DisplayAdvanced, regardless of whether user declare this output as a key in Output:Table:Tabular or not.

So I was thinking if it's preferable to add a heat emission calc and report flag as well, for example, as a simulation parameter.

Reply:
@XuanLuoLBL It's fine to make the new report variables active only with DisplayAdvancedReportVariables. 

I would only add a new simulation parameter if you think the extra calculations will drag down performance.   If you do, then just one flag should be adequate - if true then do the calcs and set up the report variables. And if you do add it, then don't require DisplayAdvancedReportVariables for these.

Reply:
@mjwitte Thanks Mike. At design stage I thought extra calculation is comparably minimal so I implement it this way. But I'll double check this.

Reply:
OK, but this line should be `if (DisplayAdvancedReportVariables)` ,without !.